go/billing/models.py

```diff
@@ -120,6 +120,24 @@ def calculate_session_credit_cost(cls, session_cost, markup_percent,
             session_cost, markup_percent, context=context)
 
     @classmethod
+    def calculate_session_unit_credit_cost(
+            cls, session_length_cost, markup_percent, context=None):
+        """
+        Return the cost per session length (in credits).
+        """
+        return cls.apply_markup_and_convert_to_credits(
+            session_length_cost, markup_percent, context=context)
```

Comment:
Yes this is a bit ambiguous. The param is the cost of a session of one unit length, hence `session_length_cost`, but I think that `session_unit_cost` makes more sense.